### ISSUE TITLE
Unable to get blocks under Y0

### ISSUE THREAD

**Issue by XenonsEU** [SkriptLang/Skript#4477]
### Skript/Server Version

```
Skript Version 2.6
```

### Bug Description

Trying to loop blocks around player and broadcast them with their location
Looping above Y=0 works just fine but under it doesn't return anything
Since 1.18 introduced blocks under Y=0 skript should also support that

### Expected Behavior

blocks should be broadcasted with their location

### Steps to Reproduce

```
command /testloop:
    trigger:
        loop all blocks in radius 10 around player:
            if loop-block is not air:
                broadcast "%loop-block% - %location of loop-block%"
                
```
go bellow Y=0 to try the command

### Errors or Screenshots

No errors

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by XenonsEU** [comment]
I just realized that 2.6 was not 1.18 support
apologies for that my brain automatically connected the 2 for some reason

if this is already in works then feel free to delete this <3

**Comment by TPGamesNL** [comment]
Indeed, Skript 2.6 does not support 1.18. However, there is an unofficial version available that should work on 1.18 in the skUnity Discord https://discord.gg/0l3WlzBPKX7WNjkf /  https://discord.com/channels/135877399391764480/836220485175607306/913842052188479518

Duplicate of https://github.com/SkriptLang/Skript/issues/4324 / https://github.com/SkriptLang/Skript/issues/4466
